Makefile

```diff
@@ -34,6 +34,9 @@ clean:
 gen:
 	$(GOBIN) run ./pkg/gen/main.go
 
+gen-gas:
+	$(GOBIN) generate ./box/...
+
```

Comment:
after running an implementation in record mode this is the command a user runs to generate blog.go that contains the gas values as a go file.

Reply:
Fold it into `gen`? Too many commands to remember otherwise.

Reply:
So `gen` doesn't actually generate anything, it prints the message producer boiler plate to the console, a developer needs to copy-paste that output into the correct files and make a few minor changes (method numbers). It was meant to serve as a tool to help me keep up with specs-actors changes when things were moving around a lot and I didn't sink a lot of time into ergonomic. I don't think remembering 2 different gen commands is unreasonable if documented.